worker.sh

```diff
@@ -9,9 +9,9 @@ ROOTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd ${ROOTDIR}
 
 # Get vars from yaml
-QUEUE=$(cat ${ROOTDIR}/instance_config.yaml | grep celery_worker_queue)
+QUEUE=$(cat ${ROOTDIR}/static_config.yaml | grep celery_worker_queue)
 QUEUE=${QUEUE#*: }
-CONCUR=$(cat ${ROOTDIR}/instance_config.yaml | grep celery_threads)
+CONCUR=$(cat ${ROOTDIR}/static_config.yaml | grep celery_threads)
 CONCUR=${CONCUR#*: }
```

Comment:
This should stay in instance_config -- it's a moving number.

Reply:
@yro are we not sure how many threads there need to be?

Reply:
@yro Can you please add more details about how `celery_threads` is used? Do we have different `celery_threads` values for each worker/instance? 
What do you mean by `moving number` or `credential based`. Want to understand the flow. Thanks

Reply:
@Qubad786 Oh sure -- it's just a value that can change periodically -- for example if we need to roll the ELB that acts as a static DNS for the rabbit server. This value is different for AWS instances vs. other kinds of instances.
`celery_threads` can be set differently for different workers. We're talking about possibly doing an experiment with using Azure instances, for example. So we need to be able to change this number per instance. It's not really credential based (that was a bad choice of words on my part), but it can change per worker instance/type.